Dockerfile.openssl

```diff
@@ -6,82 +6,79 @@ ARG IPSEC_MB_VERSION="v0.54"
 ARG IPP_CRYPTO_VERSION="ippcp_2020u2"
 
```

Comment:
While you are updating the Dockerfile, try bumping up the QAT library version to `qat1.7.l.4.11.0-00001`. Are the other components the latest versions?

Reply:
@ipuustin 
> While you are updating the Dockerfile, try bumping up the QAT library version to qat1.7.l.4.11.0-00001.

Done.

> Are the other components the latest versions?

There are newer versions of IPP_CRYPTO, IPSEC_MB, QAT_ENGINE_VERSION.

IPSEC_MB, QAT builds/installs, IPP_CRYPTO, QAT_ENGINE fail, so, perhaps, let sort this updates later one by one.

Reply:
I can take a look `ipp-crypto_2020_update3`.

Reply:
@ozhuraki For ipp-crypto and QAT Engine, follow these changes: https://github.com/intel/intel-device-plugins-for-kubernetes/pull/492

Reply:
@mythi 
> For ipp-crypto and QAT Engine [...]

Done.

After QAT Engine update I don't see libqat.so anymore, but there's libqatengine.so. I included it into the final image.

Is this OK?

Reply:
The openssl config file should point to the right engine. You need to try out if things still work (maybe just by using `openssl speed` or similar).

Reply:
@ozhuraki the new engines are called `qat-sw`  and `qat-hw`. `libqatengine.so` should not be added

Reply:
@mythi 

Thanks for the clarification. Removed it.

Reply:
@ipuustin 
> The openssl config file should point to the right engine. You need to try out if things still work (maybe just by using openssl speed or similar).

I have checked, there's a problem with the new version of QAT_ENGINE:

```
# openssl speed -engine=qat-hw
invalid engine "qat-hw"
139655177307456:error:25066067:DSO support routines:dlfcn_load:could not load the shared library:crypto/dso/dso_dlfcn.c:118:filename(/usr/lib64/engines-1.1/qat-hw.so): /usr/lib64/engines-1.1/qat-hw.so: undefined symbol: qat_num_instances
[...]
```

Could anyone take a look? Or shall we, perhaps, handle library updates separately?

Reply:
Let's handle this version update separately then.